web/themes/custom/pineapple/sass/layouts/nodes/_article.scss

```diff
@@ -0,0 +1,49 @@
+.article--teaser {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+
+  @include breakpoint($desk) {
+    width: 75%;
+    flex-direction: row;
+  }
+
+  .teaser-content {
+    background-color: #f1f0f4;
+    padding: $unit * 4;
+    padding-top: 0;
+    width: 100%;
+
+    @include breakpoint($desk) {
+      height: 400px;
+    }
+
+    p {
+      color: $primary;
+    }
+
+    .teaser-link {
+      background-color: $primary;
+      padding: $unit * 2;
+      color: white;
+      display: inline-block;
+
+      &:hover,
+      &:focus {
+        text-decoration: underline;
+        background-color: $dark-grey;
+      }
+    }
```

Comment:
teaser link should be converted to a btn class or something equivalent so it can be reused else on the site

Reply:
done